docs/contents/dev/dev-write-1st-app.md

```diff
@@ -9,7 +9,7 @@ We'll use the classical [wordcount](https://github.com/apache/incubator-gearpump
 	  override val options: Array[(String, CLIOption[Any])] = Array.empty
 	
 	  override def main(akkaConf: Config, args: Array[String]): Unit = {
-	    val context = ClientContext(akkaConf)
+	    val context = RuntimeEnvironment.get().newClientContext(akkaConf)
```

Comment:
If this is a public interface, then `RuntimeEnvironment` should also provide it. If not, please make it internal.